src/hash.cr

```diff
@@ -79,6 +79,10 @@ class Hash(K, V)
     !!find_entry(key)
   end
 
+  # Fetch an environment variable corresponding to a key. Throws an exception
```

Comment:
This comment looks like just copied from `ENV.fetch`. It talks about environment variables, instead of normal hash.